tests/fpga/symbol_mapping.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+# The scope of the test is to verify that nestedSDFG mapped symbols are handled correctly
```

Comment:
Please elaborate a bit more on what should happen, and what was wrong before. You can even link the github issue if there is one (e.g., "see #12345").

Reply:
Mapping was not handled at all.